server/src/test/java/org/elasticsearch/search/aggregations/bucket/prefix/IpPrefixAggregatorTests.java

```diff
@@ -1044,7 +1054,7 @@ public void testAggregationWithQueryFilter() throws IOException {
                 ipPrefix.getBuckets().stream().sorted(IP_ADDRESS_KEY_COMPARATOR).map(InternalIpPrefix.Bucket::getDocCount).toList(),
                 List.of(4L)
             );
-        }, fieldType);
+        }, new AggTestConfig(aggregationBuilder, fieldType));
```

Comment:
I think `withQuery(query)` needs to be added here?

Reply:
yup, looks like I dropped that by accident.